.github/workflows/rust.yml

```diff
@@ -10,38 +10,31 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
-  build:
-    name: Run Build
-    runs-on: ubuntu-latest
-
-    steps:
-    - uses: actions/checkout@v2
-    - name: Build
-      run: cargo build --verbose
```

Comment:
I don't think we should remove the `Build` step completely. Also, seems like the PR workflow status is stuck because it is missing 🤣. So yeah...

But either way, I think we should keep it.

Reply:
clippy is already building the crate, and `cargo test` also builds the crate. Having *build* as additional job just increases the workflow duration without giving us any additional information.